accord-core/src/main/java/accord/topology/TopologyManager.java

```diff
@@ -70,7 +76,18 @@
  */
 public class TopologyManager
 {
-    private static final AsyncResult<Void> SUCCESS = AsyncResults.success(null);
+    private static final FutureEpoch SUCCESS;
+
+    static
+    {
+        SUCCESS = new FutureEpoch(Long.MAX_VALUE);
+        SUCCESS.future.trySuccess(null);
+    }
+
+    // How long before we start notifying waiters on an epoch of timeout,
+    private static final long EPOCH_INITIAL_TIMEOUT_MILLIS = 10_000;
+    // How often we check for timeout, and once an epoch has timed out, how often we timeout new waiters
+    private static final long WATCHDOG_INTERVAL_MILLIS = 2_000;
```

Comment:
we have a config class now, we should be pulling from there

Reply:
Will move it to `LocalConfig`